### PR TITLE
Renice background services

### PR DESCRIPTION
**Pull request by Firefishy**
Renice background service to allow full speed important services.
Reniced services will be ignored for ondemand CPU scaling saving power.

Signed-off-by: Grant Slater <git@firefishy.com>

### CODE REVIEW

File: cookbooks/rsyncd/recipes/default.rb [hunk 1]
```diff
@@ -48,7 +48,8 @@
 systemd_service "rsync-override" do
   service "rsync"
   dropin "override"
-  exec_start "/usr/bin/rsync --daemon --no-detach --bwlimit=16384"
```
Comment (tomhughes) [inline cookbooks/rsyncd/recipes/default.rb:51]: Was getting rid of the bandwidth limit intentional?

Comment (Firefishy) [reply cookbooks/rsyncd/recipes/default.rb:51]: yeah it was... Sneaking it in ;-)

Comment (Firefishy) [reply cookbooks/rsyncd/recipes/default.rb:51]: Happy to make it a separate PR if desired.

Comment (Firefishy) [reply cookbooks/rsyncd/recipes/default.rb:51]: Removed from PR.

**Pull request merged**